interfaces/functions.def

```diff
@@ -504,6 +504,13 @@ igraph_get_all_simple_paths:
         PP-R: get.all.simple.paths.pp
         IGNORE: RR
 
+igraph_spanner:
+        PARAMS: GRAPH graph, OUT GRAPH spanner, INT stretch, \
```

Comment:
This should be a `REAL` if the stretch factor can be an arbitrary number greater than 1.